e2e/testcases/marriage/1-marriage-event-declaration.spec.ts

```diff
@@ -122,6 +122,7 @@ test.describe('1. Marriage event validation', () => {
     await test.step('1.11.3. Confirm "Save & Exit" button', async () => {
       await page.getByText('Save & Exit', { exact: true }).click()
       await page.getByText('Confirm', { exact: true }).click()
+      await page.waitForTimeout(500) // because the page is shown twice
```

Comment:
Just a comment, let's try to hide these behind abstractions or not do at all just so that no one else starts using the timeouts